src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import Button from './Button';
+
+function Calculator() {
+  return (
+    <div className="calc">
+      <span className="keys4">0</span>
+      <Button text="AC" styling="key1" />
+      <Button text="+/-" styling="key1" />
```

Comment:
- [ ] In accordance with the project requirements, it is advisable to introduce a child component within the Calculator.js file. I recommend the extraction of this code into a separate component, which can subsequently be called within the Calculator.js file. By encapsulating specific functionality or UI elements into child components, it becomes easier to manage and reuse code, facilitating collaboration and future updates.😊
```suggestion
// Calculator.js (Parent Component)
import React from 'react';
import CalculatorDisplay from './CalculatorDisplay'; // Import the child component

export default function Calculator() {
  return (
    <div className="calciGrid">
      {/* Render the child component */}
      <CalculatorDisplay />
      {/* Other calculator buttons and UI elements */}
    </div>
  );
}



```suggestion
// CalculatorDisplay.js (Child Component)
import React from 'react';

function CalculatorDisplay() {
  return (
    <div className="output">
        <div className="prev-Num" />
        <div className="curr-Num">0</div>
      </div>
  );
}

export default CalculatorDisplay;

```